res/css/views/elements/_AccessibleButton.scss

```diff
@@ -60,7 +68,14 @@ limitations under the License.
     background-color: $button-danger-bg-color;
 }
 
-.mx_AccessibleButton_kind_danger.mx_AccessibleButton_disabled {
+.mx_AccessibleButton_kind_danger_outline {
+    color: $button-danger-bg-color;
+    background-color: $button-secondary-bg-color;
+    border: 1px solid $button-danger-bg-color;
+}
+
```

Comment:
Why are we introducing new button types here, and what's a `primary_outline`?

Reply:
it's part of my eternal quest to put all our button css through the `kind` property instead of having to apply a splattering of classes.

It's a primary button that has an outline instead of a fill.

Reply:
Ah I see - but when used it appears alongside a 'continue' button whose default kind is `primary`? Shouldn't there generally just be one primary button? Why is the cancel button not a normal cancel button?

Reply:
primary describes the colour scheme, not the action it takes. We have a number of primary buttons all over settings which violate the theory of having one primary button.

Reply:
The cancel button is different because I couldn't find a unifying style for any of this that didn't have a boatload of assumptions as part of it.

Reply:
I think my understanding was that the outline buttons were just normal buttons, but OK.